en_ewt-ud-train.conllu

```diff
@@ -34056,7 +34056,7 @@
 8	2004	2004	NUM	CD	NumForm=Digit|NumType=Card	5	nummod	5:nummod	SpaceAfter=No
 9	,	,	PUNCT	,	_	14	punct	14:punct	_
 10	the	the	DET	DT	Definite=Def|PronType=Art	13	det	13:det	_
-11	9/11	9/11	PROPN	NNP	Number=Sing	12	compound	12:compound	_
+11	9/11	9/11	NUM	CD	NumForm=Digit|NumType=Card	12	nummod	12:nummod	_
```

Comment:
compound, not nummod. nummod is limited to quantity modification.

Reply:
And let's keep the XPOS as `NNP`. PTB uses different criteria for proper nouns (treating any content word in a proper name as a proper noun, which does apply here).

Reply:
```
# sent_id = weblog-blogspot.com_dakbangla_20050311135387_ENG_20050311_135387-0092
```

9/11 plotters is an nmod with CD as the xpos, and I would argue that 9/11 is a proper noun in that situation as well.

```
# sent_id = weblog-blogspot.com_dakbangla_20050311135387_ENG_20050311_135387-0100
```

After 9/11 ... 9/11 is tagged NUM/CD but is once again being used as a proper noun

test set
```
# sent_id = weblog-juancole.com_juancole_20040722101300_ENG_20040722_101300-0018
```

9/11 hijackers: 9/11 is a nummod of hijackers, once again in a context where 9/11 could be interpreted as a proper noun but is tagged CD.

anyway this is a bit of evidence for two problems: scope creep on the mission of "adding NumForm to words which don't have NumForm", and sending a programmer to do a linguist's job :P

Reply:
Ha. Arguably the PTB annotators were not consistent about the tag! But I think the XPOS should be left alone for now rather than changed from `NNP` to `CD`.

As for the inconsistent deprels, `compound` is correct in my judgment (keep in mind that the ones that were `nummod` may have been automatically converted). But I can work on that issue later.

Reply:
Looking over the docs, I agree that `compound` is better than `nummod`, so I changed it everywhere for 9/11.

I could change 9/11 to NNP everywhere, or change it to CD everywhere, but leaving it in a mixed state seems weird.  There are many other cases where non-PROPN upos is converted to NNP in the xpos, so perhaps NNP is better?

Reply:
I just pushed a change which also makes 9/11 NUM/NNP everywhere.  LMK if that looks better

Reply:
I think it's debatable. On some level "9/11" by itself is just a date, so NNP in "9/11 plot" would be inconsistent with other dates that are CD, but within "9/11 Commission" it's part of a larger proper name hence NNP. But of course "9/11" is not just any date but a date with special significance, so NNP across the board could reflect that?

Reply:
I can also just leave them all as CD if not part of Commission, NNP when part of Commission.  I don't think that's horrible

Reply:
I think that's safest (I don't have easy access to other Penn corpora right now to compare what is done there).